build.sbt

```diff
@@ -154,6 +156,16 @@ lazy val `cluster-bootstrap` = project
   )
   .dependsOn(`akka-management`)
 
+lazy val `rolling-update-kubernetes` = project
+  .in(file("rolling-update-kubernetes"))
+  .enablePlugins(AutomateHeaderPlugin)
+  .settings(
+    name := "akka-rolling-update-kubernetes",
+    libraryDependencies := Dependencies.RollingUpdateKubernetes,
+    mimaPreviousArtifactsSet
+  )
+  .dependsOn(`akka-management`, `akka-management-pki`, `cluster-bootstrap`)
+
```

Comment:
I was wondering if maybe this could be only `rolling-update`.. not sure if we need to tie it to kubernetes on the depedency name. In the sense that maybe other things related with rolling update in general can be fit here in the future? kubernetes specific things are under a specific `package` anyway. just opening up the discussion...